src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ListViewItem.ListViewItemBaseAccessibleObjectTests.cs

```diff
@@ -336,5 +336,18 @@ public void ListViewItemBaseAccessibleObject_Toggle_WorksExpected(bool isChecked
             Assert.Equal((UiaCore.ToggleState)expected, accessibleObject.ToggleState);
             Assert.False(control.IsHandleCreated);
         }
+
+        [WinFormsFact]
+        public void ListViewItemBaseAccessibleObject_GetPropertyValue_AutomationId_ReturnsExpected()
+        {
+            using ListView listView = new();
+            ListViewItem item = new();
+            listView.Items.Add(item);
+            ListViewItemBaseAccessibleObject accessibleObject = (ListViewItemBaseAccessibleObject)item.AccessibilityObject;
```

Comment:
It's appropriate to use var in this case because of the explicit cast.
```suggestion
            var accessibleObject = (ListViewItemBaseAccessibleObject)item.AccessibilityObject;
```